tools/RAiDER/statsPlot.py

```diff
@@ -527,17 +550,25 @@ def plot_variogram(self, gridID, timeslice, coords, workdir='./', d_test=None, v
                 timeslice[6:8]), int(timeslice[-4:-2]), int(timeslice[-2:]))
 
         if dists is not None and vario is not None:
+            # TODO temp fix first multiply by 1,010,000 m as crude conversion from degrees to km
+            dists = [i*1010000 for i in dists]
+            dists = [convert_SI(i, 'm', 'km') for i in dists]
             plt.scatter(dists, vario, s=1, facecolor='0.5', label='raw')
         if dists_binned is not None and vario_binned is not None:
+            # TODO temp fix first multiply by 1,010,000 m as crude conversion from degrees to km
+            dists_binned = [i*1010000 for i in dists_binned]
+            dists_binned = [convert_SI(i, 'm', 'km') for i in dists_binned]
             plt.plot(dists_binned, vario_binned, 'bo', label='binned')
         if res_robust is not None:
             plt.axhline(y=res_robust[1], color='g',
                         linestyle='--', label='ɣ\u0332\u00b2({}\u00b2)'.format(self.unit))
             plt.axvline(x=res_robust[0], color='c',
                         linestyle='--', label='h\u0332(°)')
         if d_test is not None and v_test is not None:
+            # TODO temp fix first multiply by 1,010,000 m as crude conversion from degrees to km
+            d_test = [i*1010000 for i in d_test]
             plt.plot(d_test, v_test, 'r-', label='experimental fit')
-        plt.xlabel('Distance (°)')
+        plt.xlabel('Distance (km)')
```

Comment:
Then in future when a user ask for a different unit. We would change units from SI to requested unit just before making the plot.

i.e. assume all units up to just before plotting to be in SI standard units. 
Then change it just before calling the plotting.